extension/app/ts/components/pages/ChangeChain.tsx

```diff
@@ -16,39 +18,37 @@ export function ChangeChain() {
 	const [connectAnyway, setConnectAnyway] = useState<boolean>(false)
 
 	useEffect( () => {
-		function popupMessageListener(msg: unknown) {
-			console.log('popup message')
-			console.log(msg)
-			fetch()
+		async function popupMessageListener(msg: unknown) {
```

Comment:
this can be declared outside `useEffect`?

Reply:
it can be declared outside, but its not used anywhere else. Do you feel its cleaner if its declared outside?

Reply:
I personally have a preference for declaring things as locally as possible.  I find it helps me keep track of where stuff are used and ensures they get deleted when they aren't used anymore (though TS mostly handles this for up to file local declarations).